dpti/workflows/analyze/compare_ti_plot.py

```
@@ -0,0 +1,582 @@
+            fe_cross = f_phase1(t_cross)
+
+            plt.plot(t_cross, fe_cross, "go", markersize=10, label="Transition point")
+            plt.annotate(
+                f"T = {t_cross:.1f}K\nG = {fe_cross:.6f}eV",
+                xy=(t_cross, fe_cross),
+                xytext=(20, 20),
+                textcoords="offset points",
+                bbox={"boxstyle": "round,pad=0.5", "fc": "yellow", "alpha": 0.5},
+                arrowprops={"arrowstyle": "->"},
+            )
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)